src/components/Button/index.js

```diff
@@ -8,24 +8,31 @@ const Button = ({
   children,
   className,
   disabled,
-  onClick,
+  emphasized,
+  href,
   id,
+  onClick,
+  ...attributes
 } = {}) => {
-  let classes = classNames('resin-control', className);
+  let classes = classNames('resin-control', 'resin-control-btn', className, { emphasized });
 
   if (btnStyle) {
-    classes += ` resin-control_btn-${btnStyle}`;
+    classes += ` resin-control-btn-${btnStyle}`;
   }
 
+  const Tag = href ? 'a' : 'button';
```

Comment:
We can leave this for later but we definitely need a good approach to dealing with links. I've seen some libraries allow let you pass a component that acts as a wrapper around the link. But I'm not sure what the best approach is here. 

Let's bring it up in the FR call?